src/routes/RemediationDetails.js

```diff
@@ -143,11 +145,15 @@ class RemediationDetails extends Component {
                         <LevelItem>
                             <Split gutter="md">
                                 { this.state.isEntitled &&
-                                    <SplitItem>
-                                        <ExecutePlaybookButton
-                                            remediationId={ remediation.id }>
-                                        </ExecutePlaybookButton>
-                                    </SplitItem>
+                                    <PermissionContext.Consumer>
```

Comment:
Can we add the `<DeniedState/>` logic all routes (meaning here)? I.e. if the read permission is taken away from me `/remediations` gets me to `<DeniedState/>` but so should `/remediations/12345` (e.g. an old bookmark)